docs/framework/index.yml

```diff
@@ -0,0 +1,132 @@
+# Start card title with a verb
+  # Card (optional)
+  - title: Learn about .NET Framework
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: .NET Framework overview
+            url: get-started/overview.md
+          - text: Migrate from .NET Framework to .NET Core
+            url: ../core/porting/index.md
+      - linkListType: reference
+        links:
+          - text: System requirements
+            url: get-started/overview.md
+          - text: .NET Framework & Windows OS versions 
+            url: migration-guide/versions-and-dependencies.md
+          - text: Determine which versions are installed 
+            url: migration-guide/how-to-determine-which-versions-are-installed.md
+          - text: Older versions of .NET Framework
+            url: /previous-versions/dotnet/
+      - linkListType: concept
+        links:
+          - text: Deployment guide for developers
+            url: deployment/deployment-guide-for-developers.md
+
+  # Card
+  - title: Install .NET Framework
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Installation guide 
+            url: install/index.md
+      - linkListType: how-to-guide
+        links:
+          - text: Install .NET Framework on Windows 10 
+            url: install/on-windows-10.md
+          - text: Install .NET Framework 3.5
+            url: install/dotnet-35-windows-10.md
+          - text: Install the developer pack or redistributable 
+            url: install/guide-for-developers.md
+          - text: Troubleshoot blocked installations
+            url: install/troubleshoot-blocked-installations-and-uninstallations.md
+
+  # Card
+  - title: Create Windows service apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Introduction to Windows service apps
+            url: windows-services/introduction-to-windows-service-applications.md
+      - linkListType: tutorial
+        links:
+          - text: Create a Windows service app 
+            url: windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md
+      - linkListType: how-to-guide
+        links:
+          - text: Install and uninstall Windows services 
+            url: windows-services/how-to-install-and-uninstall-services.md
+
+  # Card
+  - title: Create WPF apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of WPF
+            url: wpf/index.md
+      - linkListType: tutorial
+        links:
+          - text: Create your first WPF app in Visual Studio
+            url: wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md
+
+  # Card
+  - title: Create Windows Forms apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of Windows Forms
+            url: winforms/index.md
+      - linkListType: how-to-guide
+        links:
```

Comment:
These seem too narrowly focused for landing page links.  Is there no getting starting tutorial that we can link to, like the Create WPF Apps card has?